src/Services/Identity/Identity.API/Identity.API.csproj

```diff
@@ -42,7 +42,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="Certificate\idsrv3test.pfx" />
+    <EmbeddedResource Remove="Certificate\idsrv3test.pfx" />
```

Comment:
I am not sure about this one. @programle  thoughts?

Reply:
We should not remove this. The cert has been added to repo, so we should revert this change